synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1128,11 +1092,8 @@ CanvasView::create_top_toolbar()
 	displaybar->append(*create_tool_separator());
 
 	{ // Refresh button
-		Gtk::Image *icon = Gtk::manage(new Gtk::Image(Gtk::StockID("gtk-refresh"), iconsize));
-		icon->show();
-
-		refreshbutton = Gtk::manage(new class Gtk::ToolButton());
-		refreshbutton->set_icon_widget(*icon);
+		refreshbutton = Gtk::manage(new Gtk::ToolButton());
+		refreshbutton->set_icon_name("gtk-refresh");
```

Comment:
`gtk-refresh` -> `view-refresh`

Reply:
I just found out orientations about it XD
https://docs.google.com/document/d/1KCVPoYQBqMbDP11tHPpjW6uaEHrvLUmcDPqKAppCY8o/pub
https://docs.google.com/spreadsheets/d/1HavJQRPpMuq-N0GoN1wJR-9KEGXpKy3-NEPpZZkUGJY/pub?output=html

when reading actiongroup here https://gitlab.xfce.org/apps/mousepad/-/issues/58

But I wouldn't have noticed until you pointed. thank you!

Reply:
done